controllers/topicPage/TopicPage.js

```diff
@@ -37,7 +38,8 @@ class TopicPage {
       payload.listBlockDomain,
       payload.listPostAnonymous
     ).getHasBlock(topicPages);
-    results.data = await filterFeeds(payload.userId, newTopicPagesWithBlock, payload.id);
+    const threshold = ACTIVITY_THRESHOLD.TOPIC_FEED;
+    results.data = await filterFeeds(payload.userId, newTopicPagesWithBlock, payload.id, threshold);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `filterFeeds` function now accepts an additional parameter `threshold`. This is a good change as it allows for more flexible feed filtering based on activity thresholds. However, ensure that all calls to `filterFeeds` throughout the codebase are updated to include this new parameter where necessary.
```diff
- results.data = await filterFeeds(payload.userId, newTopicPagesWithBlock, payload.id);
+ const threshold = ACTIVITY_THRESHOLD.TOPIC_FEED;
+ results.data = await filterFeeds(payload.userId, newTopicPagesWithBlock, payload.id, threshold);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->